cvat/apps/engine/models.py

```diff
@@ -457,15 +457,37 @@ def get_tmp_dirname(self) -> str:
 
 
 @transaction.atomic(savepoint=False)
-def clear_annotations_in_jobs(job_ids: Iterable[int]):
+def clear_annotations_in_jobs(job_ids: Iterable[int], attrspec_ids: models.QuerySet[list[int]]):
```

Comment:
```suggestion
def clear_annotations_in_jobs(job_ids: Iterable[int], *, attrspec_ids: models.QuerySet[list[int]]):
```

I think making this parameter kw-only will make function calls more clear